base/src/plugin-manager/plugin-manager.ts

```diff
@@ -63,10 +63,31 @@ export class PluginManager {
               try {
                 var result = JSON.parse(this.responseText);
                 let allPlugins = PluginManager.parsePluginDefinitions(result);
+                const searchParams = new URLSearchParams(window.location.search);
+                const useV2Desktop = searchParams.has("use-v2-desktop") && (searchParams.get("use-v2-desktop") == '1');
+                let validPlugins = allPlugins.filter((plugin) => {
+                  if (plugin.type != 'application') {
```

Comment:
should be plugin?.type if parsePluginDefinitions(result) method returns an array that has undefined values

Reply:
preferably with an error log in case plugin response text could not be parsed (may be done already upstream in PluginManager)

Reply:
if you dont have a plugin type, you'll fail startup.